bindings/nodejs/package.json

```diff
@@ -49,9 +47,23 @@
     "build:debug": "napi build --platform --js generated.js --js-package-name @opendal/lib",
     "docs": "typedoc",
     "prepublishOnly": "napi prepublish -t npm",
+    "format": "run-p format:md format:json format:yaml format:source format:rs",
+    "format:md": "prettier --parser markdown --write './**/*.md'",
+    "format:json": "prettier --parser json --write './**/*.json'",
+    "format:rs": "cargo fmt",
+    "format:source": "prettier --config ./package.json --write './**/*.{js,ts,mjs}'",
+    "format:yaml": "prettier --parser yaml --write './**/*.{yml,yaml}'",
```

Comment:
I prefer fewer tasks, and a single format command is sufficient for me.

Reply:
Merged as a single one.